src/ch341a.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * avrdude - A Downloader/Uploader for AVR device programmers
+ *
+ * avrdude support for CH341
```

Comment:
Is that really the general CH341 family or just CH341A?

Reply:
It is mainly CH341A and CH341B. CH341F may work as well. CH341C and CH341T will not work as they do not support SPI. CH341H may work but it is obsolete.
However, the inf from WCH refers them as `CH341A` so I will prefer to keep it as CH341A. 
* https://github.com/avrdudes/avrdude/pull/1215#issuecomment-1483719777